src/tagmanager/tm_tag.c

```diff
@@ -693,6 +693,47 @@ const gchar *tm_tag_context_separator(TMParserType lang)
 	}
 }
 
+gboolean tm_tag_has_full_context(TMParserType lang)
```

Comment:
I think this is a misnomer, since it doesn't operate on a tag. I think it would fit better in tm_parser.c, as `tm_parser_has_full_context()`

Reply:
That's a good point. Since we now have tm_parser.c/h files where also the tag mappings are present, it seems to be a good place for this function too. And tm_tag_context_separator(), tm_tag_langs_compatible() should be moved there too so all these parser-specific mappings are at one place.

Reply:
Yes, it makes more sense to have them in tm_parser.  I actually thought so, but placed it next to the similar functions we already had.
Now I added a commit moving them all to tm_parser.